### ISSUE TITLE
speed climbing damage

### ISSUE THREAD

**Issue by OgelGames** [pandorabox-io/pandorabox.io#505]
Climbing down a climbable node (beacon beam, ladder, etc), and landing on a node, while wearing lava boots and with beacon speed boost lv4 effect, causes damage similar to fall damage.

Side note: jihuu and chi both died from this bug, and lost all their items due to bones not being placed and items despawning (i think), including jihuu's mod armor.

**Comment by BuckarooBanzay** [comment]
> Climbing down a climbable node (beacon beam, ladder, etc), and landing on a node, while wearing lava boots and with beacon speed boost lv4 effect, causes damage similar to fall damage.

Don't know what to think of this, but *some* rules of physics should still apply: climbing fast down is the same as falling, right? (in terms of speed and deceleration)

Besides: i think i saw a part of that damage-check somewhere in the engine, it will be fun fixing that :smile:

**Comment by OgelGames** [comment]
> but _some_ rules of physics should still apply

True, but when experienced in-game, it doesn't _feel_ like it should happen, especially when doing the same thing while flying doesn't hurt... 😕
